### DISCUSSION TITLE
Custom admonitions in multi-instance

### DISCUSSION THREAD

**Discussion by lkrell** [facebook/docusaurus#4685] [Q&A] [answered]
I'm not sure if I am doing something wrong, need to adjust something, but my custom admonitions are not generating in multi-instance files. I noticed any customized admonitions added to the docusaurus.config.js are not generating in files outside of the /docs folder. It is only generating that content as plain text in <p> tags.

These admonitions work just fine in /docs files. Any ideas? I've been trying to figure out if I am not handling something correctly, or if this is a bug?

![image](https://user-images.githubusercontent.com/76010626/116110393-442e3800-a67b-11eb-8f15-6ac2695e48a2.png)

**Comment by slorber** [comment, marked as answer]
have you defined the custom admonitions on the config of the preset + the additional docs plugin? You have to declare it everywhere you plan to use it.

As you don't share any config or repro it's impossible for me to be sure you configured your site properly, but it looks like a config issue.

I don't know if your screenshots are the actual or expected result either, please add a comment to them or something that can help me figure things out. I'm assuming this is a screenshot of a doc outside the ./docs folder right? Because I don't know either how your custom admonitions should look like :)

**Reply by lkrell** [reply]
AHA that worked! The docs mention and have info for when using the preset, and when not using the preset, but not that you may need to add plugin/custom settings in each plugin id/multi-instance. Or I misunderstood...very possible! 🤦 

The screenshot shows the lack of generation, with code showing it's just putting them in <p> tags. It should be generating them correctly. As you mentioned, I added the admonitions to both the default and new multi-instance plugin/preset and it worked! 🚀  Thank you so much @slorber !

**Reply by slorber** [reply]
great!

I also find it a bit confusing to be able to specify the same config in the preset + the extra plugin. 
For multi-instance using no preset but 2 plugins can be less confusing maybe? 
Not sure how to document this better but if you have ideas PRs are welcome.

I'd also like to add a way to configure markdown plugins globally for the whole site, instead of adding them to each plugin instance.

**Reply by lkrell** [reply]
After I finish with our crazy doc site in my current project, I may try submitting a PR to help with documenting a bit more! :) Been keeping notes on a ton of lessons learned.
Global plugin configuration would be amazing!

**Reply by slorber** [reply]
Thanks

Let's track global config here, forget we already had an issue: https://github.com/facebook/docusaurus/issues/4625
